### DISCUSSION TITLE
Confusion about NNs training

### DISCUSSION THREAD

**Discussion by GVink42** [vdemichev/DiaNN#672] [Q&A] [answered]
Hello, 
I am trying to understand how the training and tests dataset are defined in DiaNN.
If all the precursors from all the samples are used for training, what is the test dataset ? Are they the same ?
If yes, is it the use of one epoch and that "DIA-NN can train each network on a part of the dataset, only using it to score precursors it has not been trained on" that avoids an overfitting ?
Thanks in advance for your answer !

**Comment by vdemichev** [comment, marked as answer]
There's no test dataset. Was such an option in DIA-NN 1.6, but one epoch means effectively zero overfitting, so we never used it.

Vadim

**Reply by GVink42** [reply]
Understood. Thank you for the the quick answer
